js_modules/dagit/packages/core/src/assets/AutoMaterializePolicyPage/AssetAutomaterializePolicyPage.tsx

```diff
@@ -24,92 +98,236 @@ export const AssetAutomaterializePolicyPage = () => {
             border={{side: 'right', color: Colors.KeylineGray, width: 1}}
-            style={{width: '294px'}}
+            flex={{grow: 0, direction: 'column'}}
+            style={{width: '296px'}}
           >
-            <LeftPanel />
+            <LeftPanel
+              evaluations={evaluations}
+              queryResult={queryResult}
+              paginationProps={paginationProps}
+              onSelectEvaluation={(evaluation) => {
+                setSelectedEvaluationId(evaluation.evaluationId);
+              }}
+              selectedEvaluation={selectedEvaluation}
+            />
           </Box>
-          <Box flex={{grow: 1}}>
-            <MiddlePanel />
+          <Box flex={{grow: 1}} style={{minHeight: 0}}>
+            <MiddlePanel
+              assetKey={assetKey}
+              key={selectedEvaluation?.evaluationId || ''}
+              maxMaterializationsPerMinute={maxMaterializationsPerMinute}
+            />
           </Box>
         </Box>
       </Box>
       <Box>
-        <RightPanel />
+        <RightPanel
+          assetKey={assetKey}
+          setMaxMaterializationsPerMinute={setMaxMaterializationsPerMinute}
+        />
       </Box>
-    </Box>
+    </AutomaterializePage>
   );
 };
 
-const LeftPanel = () => {
-  const evaluations = generateMockEvaluations();
-
+export const PAGE_SIZE = 30;
+function LeftPanel({
+  evaluations,
+  queryResult,
+  paginationProps,
+  onSelectEvaluation,
+  selectedEvaluation,
+}: {
+  evaluations: EvaluationType[];
+  queryResult: ReturnType<typeof useEvaluationsQueryResult>['queryResult'];
+  paginationProps: ReturnType<typeof useEvaluationsQueryResult>['paginationProps'];
+  onSelectEvaluation: (evaluation: EvaluationType) => void;
+  selectedEvaluation?: EvaluationType;
+}) {
   return (
-    <Box flex={{direction: 'column'}}>
-      {evaluations.map((evaluation) => {
-        const date = dayjs.unix(evaluation.date);
-        const formattedDate = date.format('lll');
-        return (
-          <CenterAlignedRow
-            flex={{justifyContent: 'space-between'}}
-            key={evaluation.date + ''}
-            padding={{horizontal: 24, vertical: 8}}
-            border={{side: 'bottom', width: 1, color: Colors.KeylineGray}}
-          >
-            {formattedDate}
-            {evaluation.materialized ? (
-              <Icon name="auto_materialize_policy" size={24} />
-            ) : (
-              <div
-                style={{borderRadius: '50%', height: '10px', width: '10px', color: Colors.Yellow50}}
-              />
-            )}
-          </CenterAlignedRow>
-        );
-      })}
+    <Box flex={{direction: 'column', grow: 1}} style={{overflowY: 'scroll'}}>
+      <Box
+        style={{flex: 1, minHeight: 0, overflowY: 'scroll'}}
+        flex={{grow: 1, direction: 'column'}}
+      >
+        {evaluations.map((evaluation) => {
+          const date = dayjs.unix(evaluation.timestamp);
+          const formattedDate = date.format('lll');
```

Comment:
This might be inconsistent with other timestamp rendering throughout the app. The `<Timestamp>` or `<TimestampDisplay>` component (I should do something to disambiguate those...) would be more consistent.

Reply:
We use this format here: https://github.com/dagster-io/internal/blob/356395a1617046d164bb434982bfaf7f64a73089/dagster-cloud/js_modules/cloud-admin-panel/src/components/AccountReview.tsx#L35